2020-InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -54,37 +49,58 @@
 
   private Camera camera1;
   private Camera camera2;
+  private boolean camera1Enable = false;
+  private boolean camera2Enable = false;
 
   private IMU imu;
 
+  
+
   private boolean chassisEnable = false;
   private boolean ImuEnable = false;
-  private boolean camera1Enable = false;
-  private boolean camera2Enable = false;
-
+  @Log
   private boolean robotInitialized = false;
+  private boolean powerCellEnable = false;
+  private PowerCell powerCell;
+  
+  @Config 
+  public void Enable_PowerCell(boolean enable){
+    powerCellEnable = enable;
+  }
+
+  @Config 
+  public void Enable_Camera1(boolean enable){
+    camera1Enable = enable;
+  }
+
+  @Config 
+  public void Enable_Camera2(boolean enable){
+    camera2Enable = enable;
+  }
+
   @Config
   public void Enable_IMU(boolean enable){
     ImuEnable = enable;
   }
+
   @Config.ToggleButton
   public void Enable_Chassis(boolean enable){
     chassisEnable = enable;
     System.out.println("Chassis Enable: " + chassisEnable);
     System.out.println("Enable: " + enable);
   }
-  @Config
-  public void Enable_Cameras(boolean enable, boolean enable2){
-    camera1Enable  = enable;
-    camera2Enable = enable2;
-  }
 
   /**
    * This function is run when the robot is first started up and should be
    * used for any initialization code.
    */
   @Override
   public void robotInit() {
+    
+    
+    oi = new OperatorInterface();
+    powerCell = new PowerCell();
+    Logger.configureLoggingAndConfig(this, false);
```

Comment:
Does this Logger call need to happen before you create the PowerCell object?

Reply:
(it looks like there might be another call to `Logger.configureLoggingAndConfig` below.